api/RS.API/RS.Web/Controllers/CandidateController.cs

```diff
@@ -82,5 +102,47 @@ public IResult ApprovedForInterview(Guid candidateId)
             var approvedCandidate = _candidateManagerService.ApprovedForInterview(candidateId);
             return approvedCandidate;
         }
+
+        private string GetExtension(IFormFile file)
+        {
+            string result = null;
+            if (file.Length > 0)
+            {
+                var ext = new List<string> { ".pdf", ".doc", ".docx" };
+                var extension = Path.GetExtension(file.FileName);
+                if (ext.Contains(extension))
+                {
+                    result = extension;
+                }
+            }
+            return result;
+        }
+
+        private void SaveFile(IFormFile file)
+        {
+            string path = Path.Combine(_hostingEnvironment.ContentRootPath, "uploadFiles");
+            if (!Directory.Exists(path))
+            {
+                Directory.CreateDirectory(path);
+            }
+            var ext = new List<string> { ".pdf", ".doc", ".docx" };
```

Comment:
move extension under api project .json file, in future these extension might be increased or decreased

Reply:
Done